cypress/support/Pages/typePage.js

```diff
@@ -0,0 +1,28 @@
+import { checkCurrentURL } from "../Utils/assertions";
+
+import { typePageLocators } from "../locators";
+
+class TypePage {
+  visit() {
+    return cy.visit(typePageLocators.url);
+  }
+
+  checkURL() {
+    return cy
```

Comment:
`return` można tutaj usunąć we wszystkich metodach

Reply:
juz zmieniam :)